pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
well i misspoke slightly, but you still want to look where i pointed because we do dump out the urls of the webhooks.  Adding the value of this field to that output seems reasonable to me.

@jwforres probably needs to be aware we're adding this too.